dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/wrapper/MockClusterInvoker.java

```diff
@@ -86,12 +86,12 @@ public Result invoke(Invocation invocation) throws RpcException {
             } catch (RpcException e) {
                 if (e.isBiz()) {
                     throw e;
-                } else {
-                    if (logger.isWarnEnabled()) {
-                        logger.warn("fail-mock: " + invocation.getMethodName() + " fail-mock enabled , url : " + directory.getUrl(), e);
-                    }
-                    result = doMockInvoke(invocation, e);
                 }
+                
+                if (logger.isWarnEnabled()) {

```

Comment:
I don't think this has some differences with the old way..

Reply:
yes，there is no functional difference，I think it's a little bit cleaner to take this logic out

Reply:
OK, that is all right.
I will approve this pr.